assets/scss/_footer.scss

```diff
@@ -0,0 +1,32 @@
+.footer{
+    background-image: url("../images/footerbg.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.2rem 3rem 1.7rem;
+
+    .icons {
```

Comment:
you don't need to nest icons and social medial inside footer, they can be a block (**B**EM) themself :)

Reply:
changed